docs/diagrams/AddMeetingCommandSequence.puml

```diff
@@ -0,0 +1,45 @@
+@startuml
+hide footbox
+actor user
+
+
+user -> ":Ui" ++ :"add_meeting t/meeting1 d/Friday st/1230 et/1330 m/online"
+create ":Parser"
+":Ui" -> ":Parser" ++: new Parser("add_meeting t/meeting1 d/Friday st/1230 et/1330 m/online")
+":Parser" --> ":Ui" --: parser:Parser
+":Ui" -> ":Parser" ++: parseCommand("add_meeting t/meeting1 d/Friday st/1230 et/1330 m/online")
+create ":AddMeetingCommand"
+":Parser" -> ":AddMeetingCommand" ++: new AddMeetingCommand("meeting1": String,"Friday":String,1230:int,1330:int,"online":String)
```

Comment:
Would like to echo above comment that the font size looks a bit small. Consider using more descriptive statements here instead of writing the whole function call, as the function call is very long and stretches the diagram horizontally?